### PR TITLE
DVCI-195: Reinstate Privacy Policy

### PR DESCRIPTION
**Pull request by mickohanlon23**
This PR adds the privacy policy back to our application:

- Add `/privacy` route and "Privacy" button in footer.
- Edits were made to the privacy policy language in accordance with our completed Privacy Review.
- Language about Google Analytics tracking was replaced with language regarding Clicky tracking to align with https://github.com/dvci/shc-web-verifier/pull/71.

### CODE REVIEW

**Review by sarahmcdougall — CHANGES_REQUESTED** [review]

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -68,32 +68,40 @@
...
+      },
+      {
+        "header": "INFORMATION WE COLLECT AUTOMATICALLY",
+        "content": "We use Clicky, a third-party web analytics service, to help us understand and analyze how the Verifier web application is used. This service is configured so that it does not use cookies, which are data that may be sent to your web browser and stored on your computer.<br/>For more information on how Clicky uses data collected through the Verifier, visit: <a target=\"top\" href=\"https://clicky.com/terms/privacy\" class=\"dotcom-only\">https://clicky.com/terms/privacy</a>. To opt out of Clicky, visit the <a target=\"top\" href=\"https://clicky.com/optout\" class=\"dotcom-only\">Clicky Opt-out page</a>."
```
[the remark this answers is not in the document]
Comment (mickohanlon23) [reply public/locales/en/translation.json:80]: What about:

> "We use Clicky, a third-party web analytics service, to help track the number of visitors to the Verifier web application in a privacy-preserving way."

[the remark this answers is not in the document]
Comment (mickohanlon23) [reply public/locales/en/translation.json:80]: Updated: https://github.com/dvci/shc-web-verifier/pull/72/commits/ddefd695556e3f75910cc1db41c3dc81a7d18e42

[the remark this answers is not in the document]
Comment (sarahmcdougall) [reply public/locales/en/translation.json:80]: I think that sounds good!

[the remark this answers is not in the document]
Comment (mickohanlon23) [reply public/locales/en/translation.json]: Great, I made the update here: https://github.com/dvci/shc-web-verifier/pull/72/commits/c4a277b21502007ddd932c4f35aefd31bf691658

[the remark this answers is not in the document]
Comment (mickohanlon23) [reply public/locales/en/translation.json]: > I like the inclusion of the link to opt out of Clicky. Should we try testing this by having someone log into our Clicky page and have someone else opt out of Clicky and then open the web verifier to check that their visit doesn't get tracked?

@sarahmcdougall confirmed that the Opt-out feature works, so I'll resolve this thread, but feel free to re-open if there are issues.

**Review by sarahmcdougall — APPROVED** [review]
I think this needs a rebase since we updated the `translation.json`, but otherwise everything seems to be working! Thanks for handling the privacy policy changes!

**Comment by mickohanlon23** [comment]
I am going to double-check the final privacy policy updates we made with the Privacy team that led our review, but once that is complete I will plan to merge this.

**Comment by mickohanlon23** [comment]
I made updates based upon the final Privacy Policy edits Linda sent on 6/30. @sarahmcdougall would you mind taking a final look before I merge?

**Comment by sarahmcdougall** [comment]
> I made updates based upon the final Privacy Policy edits Linda sent on 6/30. @sarahmcdougall would you mind taking a final look before I merge?

Just read through it and it sounds good to me!

**Pull request merged**